packages/date-picker/src/DatePicker/components/DateInput/DateInput.tsx

```diff
@@ -150,7 +181,21 @@ export const DateInput: React.VFC<DateInputProps> = ({
         onFocus={handleFocus}
         {...inputProps}
       />
-
+      {status && (
+        <div
+          className={classnames(styles.message, {
+            [styles.disabled]: disabled,
+          })}
+        >
+          <FieldMessage
+            id={`${id}-field-message`}
+            automationId={`${id}-field-validation-message`}
```

Comment:
```suggestion
```

Unless someone needs them for good reason, we shouldn't be adding ids (definitely not automation ones too since they're deprecated).